src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx

```diff
@@ -4,115 +4,33 @@ import { Callout } from "/src/components/callout";
 
 <Callout emoji="⭐">
   The [Token Whitelisting for Squid Router
-  form](https://github.com/axelarnetwork/axelar-configs) is now live! Download
-  the repo, install dependencies, and `npm run wizard` on the command line to
+  form](https://github.com/axelarnetwork/axelar-configs/blob/main/cli/wizard/commands/list-squid-token/README.md) is now live! Download
+  the [`axelar-configs`](https://github.com/axelarnetwork/axelar-configs) repo, install dependencies, and `npm run wizard` on the command line to
   access the interactive form through the wizard.
 </Callout>
 
-If you already have an ERC-20 token on one or more blockchains, you can turn it into an Interchain Token by deploying [Token Managers](https://docs.axelar.dev/dev/glossary#token-manager). Token Managers can be either [Lock/Release](https://docs.axelar.dev/dev/glossary#lockunlock) or [Mint/Burn](https://docs.axelar.dev/dev/glossary#mintburn). Canonical tokens are registered under the local chain's Lock/Release token manager and mint/burn on remote chains. They can be deployed to remote chains by anyone and don't depend on a deployer address/salt.
+You can turn an ERC-20 token that exists on one or more blockchains into an Interchain Token by deploying a [token manager](/dev/send-tokens/glossary#token-manager), which can be either [Lock/Unlock](/dev/send-tokens/glossary#lockunlock) or [Mint/Burn](/dev/send-tokens/glossary#mintburn). Interchain Tokens created this way are called [canonical tokens](/dev/send-tokens/glossary#canonical-interchain-token).
 
-The Interchain Token Service is deployed to [`0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C`](https://etherscan.io/address/0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C) while the Interchain Token Factory is deployed to [`0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66`](https://etherscan.io/address/0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66).
```

Comment:
The create token page has this info, but I need those details if I am on the upgrade token page and want to link my tokens or create a bridgeable version programmatically. Can we add it here? Or link to where we have it on the testnet resource page to avoid repetition.

Reply:
Linked here to avoid making devs page-jump too much.